mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/MainActivity.java

```diff
@@ -129,6 +131,10 @@
   private static final String DINO_ASSET_TAG = "dino_asset_name";
   // All GIF animation assets and tags
   private ArrayList<Bitmap> GIFBitmaps = new ArrayList<Bitmap>();
+  private int gifCurrentIndex = 0;
+  private static final int GIF_FRAME_RATE = 15; // 15 FPS
```

Comment:
Out of curiosity, why 15fps here?  I think 20fps is usually the lowest we can go for normal video to still be considered truly "realtime", but perhaps GIFs are meant to be played slower?  Or is our system not fast enough right now, and needs the extra time?

Reply:
Normal FPS for GIFs are typically 15-24 fps, but will set to 20fps for consistency.

Reply:
[Link to standardized GIF guidelines](https://www.bluefrogdm.com/blog/best-practices-for-creating-animated-gifs#:~:text=Framerate&text=Standard%20GIFs%20run%20between%2015,lower%20the%20quality%20will%20be.)